plutus-core/index-envs/src/Data/DeBruijnEnv.hs

```diff
@@ -33,22 +38,23 @@ instance DeBruijnEnv (BRAL.RAList a) where
     {-# INLINABLE cons #-}
     cons = BRAL.Cons
     {-# INLINABLE index #-}
-    index = BRAL.safeIndex
+    index e i = BRAL.safeIndexZero e (i-1)
```

Comment:
you were going to try using the `*IndexOne` functions in a follow up, right? Maybe make a ticket so we don't forget.

Reply:
y followup